explore/src/main/scala/explore/targeteditor/BrightnessesEditor.scala

```diff
@@ -138,9 +144,9 @@ sealed abstract class BrightnessesEditorBuilder[T, Props <: BrightnessesEditor[T
                       onClick = brightnesses.mod(_ - cell.value)
                     )(Icons.Trash)
                   ),
-                size = 20,
-                minSize = 20,
-                maxSize = 20,
+                size = 20.toPx,
+                minSize = 20.toPx,
+                maxSize = 20.toPx,
```

Comment:
scalajs react has a `.px` syntax, could we reuse it?

Reply:
`.px` returns a `String`, whereas here we need the numeric value.

Actually, I named this `px` initially, but it clashed with `scalajs-react`'s.